lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,19 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.last.nil?
+            ""
+          else
+            "#{comments.last.body.inspect}"
+          end
+        end
+
+        def approved?
```

Comment:
Man, nevermind, I don't want to set a precedent as we'd like to keep the base-class structure in tact and the modified behavior should live separate from it.  I'm good with this :ok_hand: